modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -3808,7 +3834,7 @@ bool GDScriptAnalyzer::get_function_signature(GDScriptParser::Node *p_source, bo
 
 	Ref<Script> base_script = p_base_type.script_type;
 
-	while (base_script.is_valid() && base_script->is_valid()) {
+	while (base_script.is_valid() && base_script->has_method(function_name)) {
```

Comment:
Do you think we have a bug report for this?

Reply:
I'm not really sure. It was just something I noticed and fixed as I went.